central/models/user.go

```diff
@@ -40,9 +41,8 @@ type Identity struct {
 }
 
 type Referral struct {
-	ID      bson.ObjectId  `bson:"_id" json:"id"`
-	Code    string         `bson:"code" json:"code"`
-	Created time.Time      `bson:"created" json:"created"`
-	Used    *time.Time     `bson:"used" json:"used"`
-	UserID  *bson.ObjectId `bson:"user_id" json:"user_id"`
+	ID        bson.ObjectId `bson:"_id" json:"id"`
+	Code      string        `bson:"code" json:"code"`
+	Created   time.Time     `bson:"created" json:"created"`
+	Remaining int           `bson:"remaining" json:"remaining"`
```

Comment:
no longer a Used or UserID param on the referrals since they will be used by multiple users